tests/test_visitors/test_ast/test_naming/test_class_attributes.py

```diff
@@ -31,6 +31,30 @@ def action_method(self, request, second):
     action_method.label = 'Do action'
 """
 
+static_attribute_in_enum = """
+import enum
+
+
+class Test(enum.Enum):
+    {0} = enum.auto()
+"""
+
+static_attribute_in_django_enum_like = """
```

Comment:
I would go the other way. I would provide a template like:

```python
template = """
class MyClass({0}):
    {1}
"""
```

and then format it with bases like:
- `choices.TextChoices`
- `Enum`
- `enum.Enum`
- etc

And bodies like:
- `UPPER: str = 1`
- `UPPER = ...`
- `UPPER_NAME = UPPER_ALIAS = auto()`
- `lower_also_fine: int = 1`
- `lower_also_fine = 1`
- etc

and check that there are no violations. Then, I would check that non-enum classes cannot have `UPPER_NAMES` with changing the base class:
- `not_enum.Base`
- `metaclass=CustomMeta`
- etc

And names:
- `UPPER_NAME: int = 1`
- etc

This cases must raise a violation.

Reply:
> Then, I would check that non-enum classes cannot have UPPER_NAMES with changing the base class:

Isn't that covered already by ``test_upper_case_class_attributes``?

Reply:
Also, maybe reuse `logic.naming.enums._ENUM_LIKE_NAMES` in test cases rather that manually writing this?

```py
@pytest.mark.parametrize(
    'base_class',
    [
        'Enum',
        'enum.Enum',
        'models.IntegerChoices',
        'IntegerChoices',
        'models.TextChoices',
        'TextChoices',
        'models.Choices',
        'Choices',
    ],
)
```
Transform to 
```py
@pytest.mark.parametrize(
    'base_class',
    _ENUM_LIKE_NAMES,
)
```